src/utils/WebResource.js

```diff
@@ -0,0 +1,24 @@
+window.tacticsUtils = window.tacticsUtils || {};
+
+(function () {
+    'use strict';
+
+    tacticsUtils.getWebResourceJSON = function (url){ 
+        return new Promise(function(resolve, reject) {
```

Comment:
You don't need to return a new Promise anymore since the fetch function returns a promise already.